com.github.camel-tooling.lsp.eclipse.client.tests.ui/tests/com.github.cameltooling.lsp.ui.tests/src/com/github/cameltooling/lsp/ui/tests/PluginInstalledTest.java

```diff
@@ -0,0 +1,72 @@
+package com.github.cameltooling.lsp.ui.tests;
+
+import static org.junit.Assert.assertTrue;
+import java.util.List;
+import java.util.ArrayList;
+
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.swt.api.Shell;
+import org.eclipse.reddeer.swt.api.TableItem;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.ctab.DefaultCTabItem;
+import org.eclipse.reddeer.swt.impl.menu.ShellMenuItem;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.swt.impl.table.DefaultTable;
+import org.eclipse.reddeer.swt.impl.table.DefaultTableItem;
+import org.eclipse.reddeer.swt.impl.text.DefaultText;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ *
+ * @author fpospisi
+ */
+@RunWith(RedDeerSuite.class)
+public class PluginInstalledTest {
+
+	public static final String provider = "JBoss by Red Hat";
+	public static final String name = "Apache Camel LSP Client Extensions Plugin";
+	public static final String version = "1.0.0";
+	public static final String id = "com.github.camel-tooling.eclipse.client";
+
+	/**
+	 * Checks if LSP plugin has correct author, name, version and id.
+	 */
+	@Test
+	public void testPluginInstalled() {
+		// Open plug-ins in Help -> About.
+		new ShellMenuItem(new WorkbenchShell(), "Window", "Navigation", "Find Actions").select();
+		Shell shell = new DefaultShell("Find Actions");
+		new DefaultText(shell, 0).setText("About");
+		new DefaultTableItem().click();
+		shell = new DefaultShell("About Eclipse Platform");
+		new PushButton(shell, "Installation Details").click();
+		shell = new DefaultShell("Eclipse Platform Installation Details");
+		new DefaultCTabItem("Plug-ins").activate();
+
+		// Get only required plugin.
+		List<TableItem> plugins = new ArrayList<TableItem>();
+		for (TableItem item : new DefaultTable().getItems()) {
+			if (item.getText(4).equals(id)) {
+				plugins.add(item);
+			}
+		}
+		assertTrue(plugins.size() == 1);
+
+		TableItem plugin = plugins.get(0);
+		// Check for correct author.
+		assertTrue(plugin.getText(1).equals(provider));
+		// Check for correct name.
+		assertTrue(plugin.getText(2).equals(name));
+		// Check for correct version.
+		assertTrue(plugin.getText(3).startsWith(version));
+		// Check for correct id.
+		assertTrue(plugin.getText(4).equals(id));
+
+		// Close plug-ins page.
+		new PushButton(shell, "Close").click();
+		shell = new DefaultShell("About Eclipse Platform");
+		new PushButton(shell, "Close").click();
+	}
```

Comment:
optional: maybe we can also add e.g. testLicense which will verify that correct up to date license is present